docs/source/guidelines.rst

```diff
@@ -1,48 +1,200 @@
 .. _Guidelines:
 
-Guidelines for Developers
-*************************
+Guidelines for Gammasim-tools Developers
+****************************************
 
-This section is meant for developers.
+This section provides help and guidelines for developers of Gammasim-tools.
+If you want to contribute to Gammasim-tools, please use one of the contact points listed at the entry page of this documentation.
+In general, please take note of the `ctapipe Development Guidelines <https://cta-observatory.github.io/ctapipe/development/index.html>`_.
+Gammasim-tools follows the same `style <https://cta-observatory.github.io/ctapipe/development/style-guide.html#>`_ and `code guidelines <https://cta-observatory.github.io/ctapipe/development/code-guidelines.html>`_.
 
-Documentation
+Project setup
 =============
 
-Sphinx is used to create this documentation automatically with each merge into the master branch, see the `Github Action workflow CI-docs <https://github.com/gammasim/gammasim-tools/blob/master/.github/workflows/CI-docs.yml>`_.
+The main code repository for Gammasim-tools is on GitHub:
+
+`https://github.com/gammasim/gammasim-tools <https://github.com/gammasim/gammasim-tools>`_
+
+The main folders for developers are the `simtools <https://github.com/gammasim/gammasim-tools/tree/master/simtools>`_, `applications <https://github.com/gammasim/gammasim-tools/tree/master/applications>`_, `tests <https://github.com/gammasim/gammasim-tools/tree/master/tests>`_, and `docs <https://github.com/gammasim/gammasim-tools/tree/master/docs>`_ folders.
+
+
+Python version
+==============
+
+Gammasim-tools is currently developed for Python 3.9.
+
+
+Code formatting
+===============
 
-For testing of Sphinx:
+Linting and code checks are done automatically using a CI workflow using ``pyflakes`` (see this `GitHub action workflow <https://github.com/gammasim/gammasim-tools/blob/master/.github/workflows/CI-unittests.yml>`_) and the pre-commit functionality.
```

Comment:
The CI unit tests don't really do code checks, right? I think I would mention just the pre-commit functionality (mentioning also black, sort, etc.) and then mention that as part of the CI also Codacy checks code formatting.